src/app/static/package.json

```diff
@@ -9,9 +9,9 @@
     "build": "gulp copy-templates && gulp styles && webpack",
     "test": "jest -c jest.config.js",
     "integration-test": "jest --testPathIgnorePatterns adr -c jest.integration.config.js",
-    "adr-integration-test": "jest --testPathPattern adr -c jest.integration.config.js",
+    "adr-integration-test": "jest --testPathPattern adr -c jest.integration.adr.config.js",
```

Comment:
(could move the `--testPathPattern`/`--testPathIgnorePatterns` flags into the config files now that they've been separated)

Reply:
This makes sense, but because the configs build on each other, and because there's no equivalent of testPathPattern in config (so I think you'd need to use testRegex instead), you'd end up with the config overriding each other in a couple of ways, which I think would be more confusing that this is...